### ISSUE TITLE
Audit trail of all viewing or touching the database

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/rcpch-audit-engine#32]
```
    #TODO Audit trail of all viewing or touching the database
```

https://github.com/rcpch/rcpch-audit-engine/blob/6c6f1745f166c6ab7e0dececdd70983822458b54/epilepsy12/view_folder/case_views.py#L23

**Comment by eatyourpeas** [comment]
This fixed with PR #76

**Comment by pacharanero** [comment]
Use django audit trail library
